src/views/vwArticle/Detail.ejs

```diff
@@ -1,79 +1,83 @@
-			<div class="card-body">Comments</div>
-		</div>
-	</div>
-	<!-- Side articles -->
-	<div class="side-content w-25">
-		<div class="card">
-			<div class="card-header"><h3>Related articles</h3></div>
-			<div class="card-body">
-				<% for (let i = 0; i < 3; i++) { %>
-				<div class="side-article d-flex flex-row mb-2">
-					<img src="https://placehold.co/150x75" alt="" class="mr-2" />
-					<p class="font-weight-bold">Side article # <%= i %> Lorem ipsum dolor sit amet.</p>
-				</div>
-				<% } %>
-			</div>
-		</div>
-	</div>
+    <div class="article-detail w-75 mr-5">
+        <div class="article-card card">
+            <div class="card-header">
+                <h1><%= article.title %></h1>
+                <p class="text-muted">
+                    <span class="mr-2"> 
+                        <i class="bi bi-calendar-event-fill"></i> 
+                        <%= new Date(article.published_date).toLocaleDateString('vi-VN') %> 
+                    </span>
+                    <span> 
+                        <i class="bi bi-eye-fill"></i> 
+                        <%= article.views %> views 
+                    </span>
+                </p>
+            </div>
+            <div class="card-body d-flex flex-column">
+                <p class="lead"><%= article.abstract %></p>
+                <%- article.content %>
+                <div class="align-self-end">
+                    <strong>Author: <%= article.writer_name %></strong>
+                </div>
+            </div>
+            <div class="card-footer d-flex flex-row flex-wrap justify-content-lg-start">
+                <% if (article.tags && article.tags.length > 0) { %>
+                    <% article.tags.forEach(tag => { %>
+                    <span class="bg-secondary p-2 mr-1">
+                        <a href="/tag/<%= tag.tag_id %>" class="text-light text-decoration-none">
+                            <%= tag.tag_name %>
+                        </a>
+                    </span>
+                    <% }) %>
+                <% } %>
+            </div>
+        </div>
+
+        <div class="comment-section card mt-5">
+            <div class="card-header">
+                <h2>Comments (<%= comments.length %>)</h2>
+            </div>
+            <div class="card-body">
+                <% if (comments.length > 0) { %>
+                    <% comments.forEach(comment => { %>
+                    <div class="comment mb-3">
+                        <strong><%= comment.commenter_name %></strong>
+                        <small class="text-muted">
+                            <%= new Date(comment.created_at).toLocaleDateString('vi-VN') %>
+                        </small>
+                        <p><%= comment.content %></p>
+                    </div>
+                    <% }) %>
+                <% } else { %>
+                    <p>No comments yet.</p>
+                <% } %>
+            </div>
+        </div>
+    </div>
+
+    <div class="side-content w-25">
```

Comment:
Check view/partials/article_xx.ejs where xx is "m", "sl", "sr", "xs"

Use those article card instead.

See how to use it in vwHomepage/test.ejs

Or <host>:<port>/homepage/art-card